dev/breeze/pyproject.toml

```
@@ -69,7 +69,7 @@ dependencies = [
     "pyyaml>=6.0.2",
     "requests>=2.31.0",
     "restructuredtext-lint>=1.4.0",
-    "rich-click>=1.7.1",
+    "rich-click>=1.8.9",
```

Should we try something like 

```suggestion
    "rich-click>=1.7.1, !=1.8.9",
```

---

But we need `1.8.9`. I think your suggestion excludes `1.8.9`

---

right, but I'm still a bit confused what's happening 🤔

---

~~could you please try `breeze ci-image build --python 3.9` and then `breeze --help` and see how it works?~~

let me try to reset breeze and see how it works

---

I am also running `uv tool install -e ./dev/breeze --force` on main once.

---

It was failing for 1.7.1 for me